services/openshift-service-impl/src/main/java/org/kontinuity/catapult/service/openshift/package-info.java

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Not EPL! :)  And not George's author tag either, so probably this file's diffs need to be killed?